dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/report/TrackerErrorCode.java

```diff
@@ -141,6 +141,8 @@
     E1309( "Generated by program rule (`{0}`) - Unable to assign value to attribute `{1}`. " +
         "The provided value must be empty or match the calculated value `{2}`" ),
     E1310( "Generated by program rule (`{0}`) - Attribute `{1}` is being replaced in tei `{2}`" ),
+    E1311( "Event: `{0}`, is referral event and needs to have at least one complete relationship" ),
```

Comment:
Could you please write it as a full sentence including punctuation like

"Event: `{0}` is a referral event that needs to have at least one complete relationship."

Same for the other error message.

Reply:
resolved